### PR TITLE
Dunify note.0.0.2

### PR DESCRIPTION
**Pull request by dinosaure**
/cc @NathanReb, @Leonidas-from-XIV retry without the GitHub redirection...

### CODE REVIEW

File: src/dune [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(library
+ (name note)
+ (public_name note)
+ (flags :standard -w -27-32-50)
```
Comment (Leonidas-from-XIV) [inline src/dune]: Please do not modify the warnings, we're trying to mess with the code as little as possible and for the installation with opam/opam-monorepo the code will be installed in release profile anyway.

**Review by Leonidas-from-XIV — COMMENTED** [review]
No need to merge this into `master` since `master` is the upstream master version and we release the port tarballs from the `duniverse` branches.

Besides this the port looks solid, just one comment.

**Review by NathanReb — COMMENTED** [review]
It looks great! Thanks for taking care of this!

I agree with @Leonidas-from-XIV, I think it's best to leave those warnings around and keep the dune port minimal! Besides that it's good to go!

**Comment by dinosaure** [comment]
I close this PR and let the branch, I will do a release soon (and I pushed-force your review).

**Pull request closed without merging**